packages/config/test/refresh-on-response.integration.spec.ts

```
@@ -0,0 +1,72 @@
+import "../src/index.js";
+
+import {constant, DITest, inject, logger} from "@tsed/di";
+import {$asyncEmit} from "@tsed/hooks";
+
+import {CONFIG_SOURCES} from "../src/constants/constants.js";
+import {type ConfigSource, withOptions} from "../src/index.js";
+
+class TestConfigSource implements ConfigSource {
+  options: {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)